source/server/server.cc

```diff
@@ -154,6 +155,7 @@ void InstanceImpl::flushStats() {
     server_stats_->total_connections_.set(numConnections() + info.num_connections_);
     server_stats_->days_until_first_cert_expiring_.set(
         sslContextManager().daysUntilFirstCertExpires());
+    ENVOY_LOG(debug, "flush metrics to sinks");
```

Comment:
again not sure what the purpose is here, but if you want to add, change this to "flushing metrics to sinks"

Reply:
I would delete this one and leave the other one.

/wait